textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
@zuozhi .. I have written a comment inside TestConstants.java about why it is inside src/main/java. It is being used by textdb-dataflow and textdb-storage projects. The classes present in src/test/java are not exposed to outside projects, when maven build occurs. It might work in local workspace, but when maven builds all the projects it only exports the classes in src/main/java to jar.